ui-core/src/stories/Input.stories.tsx

```diff
@@ -173,6 +173,56 @@ export const ErrorInput: Story = {
   },
 };
 
+export const TooltipErrorInput: Story = {
+  args: {
+    label: 'Name',
+    type: 'text',
+    required: true,
+    value: 'Michel Sardou',
+    statusWithMessage: {
+      tooltip: 'right',
+      status: 'error',
+      message: '“Michel Sardou” can’t be used',
+    },
+    statusIconPosition: 'before-status-message',
```

Comment:
Control this props here means I can make an error when importing the component. Maybe we should have so check in osrd-ui side like "if tooltip is defined, I want to have `statusIconPosition='before-status-message'`"